packages/upload/src/FzUpload.vue

```diff
@@ -93,11 +99,16 @@ function handleDrop(event: DragEvent) {
 function handleInputChange(event: Event) {
   if (!input.value?.files) return;
   addFiles([...input.value.files]);
+  emit("fzupload:change", model.value);
```

Comment:
What is this for? I don't think it does what you want, you are emitting the old model

Reply:
addFiles assign the new model